### PR TITLE
fix: show market readonly data even no wallet connected.

### LINKED ISSUE #599: Stats don't load without Metamask Connection
**Describe the bug**

Data doesn't load when metamask is not connected.

**To Reproduce**

This is in staging, you can reproduce it there using a browser without metamask (e.g. Google Chrome Incognito)

**Expected behavior**

All the statistics for the markets should appear so visitors can still see the markets volume even without having metamask

**Screenshots**

Screen Shot 2022-03-22 at 10 29 27 AM

**Desktop (please complete the following information):**

- OS: Mac
- Browser: Google Chrome

**Additional context**

I think this has something to do with either RPC or Graph API calls. When the page loads up and I look at what network calls its making, I see its just continuously querying RPC and Graph API (see screenshot). I tried to change these RPC and Grpah APIs on my localhost but it didn't immediately fix the error. Something else must be going on.

This works ok now on app.ricochet.exchange and was previously working on staging.ricochet.exchange, which has the latest v2 app code deployed to it.

### PR DESCRIPTION
**Pull request by zhyd1997**
Fixes #599

### CODE REVIEW

File: src/store/main/sagas/sweepQueryFlow.ts [hunk 1]
```diff
@@ -27,19 +27,24 @@ export function* sweepQueryFlow() {
...
-    .forEach((stream:any) => {
-      const streamedSoFar = streamedSoFarMap[`${stream.token.id}-${stream.receiver.id}`] || 0;
-      Object.assign(streamedSoFarMap, {
-        [`${stream.token.id}-${stream.receiver.id}`]: Number(streamedSoFar) + Number(calculateStreamedSoFar(
-          stream.streamedUntilUpdatedAt,
-          stream.updatedAtTimestamp,
-          stream.currentFlowRate,
-        )),
+
+  if (address) {
```
Comment (zhyd1997) [inline src/store/main/sagas/sweepQueryFlow.ts:33]: only query streams when any wallet is connected.

Comment (0xSero) [reply src/store/main/sagas/sweepQueryFlow.ts:33]: I made a solution for this before I saw yours, which do you think is better

image

Comment (zhyd1997) [reply src/store/main/sagas/sweepQueryFlow.ts:33]: Please make sure that it works and check your browser console, thanks!

**Review by mikeghen — APPROVED** [review]

**Comment by 0xSero** [comment]
Your solution is better, thank you

**Pull request merged**